File: algo-2/latihan-haskell/bab-1/01_nilai_kuliah.hs
main :: IO ()
main = do
  putStr "input nilai harian: "
  nilaiHarian <- readLn
  putStr "input nilai tugas: "
  nilaiTugas <- readLn
  putStr "input nilai UTS: "
  nilaiUts <- readLn
  putStr "input nilai UAS: "
  nilaiUas <- readLn

  let nilaiAkhir = ((nilaiHarian / 10) + (nilaiTugas * 2 / 10) + (nilaiUts * 3 / 10) + (nilaiUas * 4 / 10))
   in putStrLn ("nilai akhir = " ++ show nilaiAkhir)

File: algo-2/latihan/bab-1/01_nilai_kuliah.py
nilai_harian = int(input("input nilai harian: "))
nilai_tugas = int(input("input nilai tugas: "))
nilai_uts = int(input("input nilai UTS: "))
nilai_uas = int(input("input nilai UAS: "))

nilai_akhir = (
    (nilai_harian * 1 / 10)
    + (nilai_tugas * 2 / 10)
    + (nilai_uts * 3 / 10)
    + (nilai_uas * 4 / 10)
)

print("nilai akhir =", nilai_akhir)
